console-subscriber/src/aggregator/mod.rs

```diff
@@ -105,12 +105,12 @@ pub(crate) struct Flush {
     pub(crate) triggered: AtomicBool,
 }
 
-// An entity that at some point in time can be closed.
-// This generally refers to spans that have been closed
-// indicating that a task, async op or a resource is not
-// in use anymore
-pub(crate) trait Closable {
-    fn closed_at(&self) -> Option<SystemTime>;
+// An entity (e.g Task, Resource) that at some point in
+// time can be dropped. This generally refers to spans that
+// have been closed indicating that a task, async op or a
+// resource is not in use anymore
+pub(crate) trait DroppedAt {
+    fn dropped_at(&self) -> Option<SystemTime>;
```

Comment:
Thanks, this is better than `Droppable` IMO.

I'm still not convinced that renaming this fields from `closed_at` to `dropped_at` is really worth such a large diff, but I'm fine with it if you think "dropped" is clearer or more accurate than "closed"!